src/odemis/driver/test/tescan_test.py

```diff
@@ -60,17 +62,27 @@
                            "focus": CONFIG_FOCUS,
                            # "camera": CONFIG_CM,
                            "pressure": CONFIG_PRESSURE},
-              "host": "192.168.1.175"
+              # "host": "192.168.1.175"
```

Comment:
why leave this here as a comment?

Reply:
Good point, changed the comment to a more descriptive one